### DISCUSSION TITLE
Auto offset shouldn't reset right away when you retry a map using quick retry

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#30677] [Ideas]
I'm used to pressing quick retry and then clicking on the auto offset button/changing if needed, but now if I quick retry I don't get to auto adjust the offset and when I press esc to retry from there, the "stats" from the last play are wiped meaning I can't auto offset anymore.

Would be nice if auto offset stored the data from the last play until you play enough of the map again or something (before it gets enough data to trigger a new auto offset "phase")

**Comment by Communified** [comment]
I agree with this, if nothing else if your second retry isn't of adequate length to have a new offset calculated, then the previous attempt's should still be there.

**Reply by xdWarmonger** [reply]
I'd even say keep the offset recommendation until you retry again.
